[2024-09-15 20:02:14,239] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: BRONZER_GET_STONCKS.BRONZE_GOOG scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-09-15 20:02:14,243] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: BRONZER_GET_STONCKS.BRONZE_GOOG scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-09-15 20:02:14,243] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-09-15 20:02:14,243] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2024-09-15 20:02:14,243] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-09-15 20:02:14,254] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): BRONZE_GOOG> on 2024-01-02 00:00:00+00:00
[2024-09-15 20:02:14,256] {standard_task_runner.py:52} INFO - Started process 11128 to run task
[2024-09-15 20:02:14,259] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'BRONZER_GET_STONCKS', 'BRONZE_GOOG', 'scheduled__2024-01-02T00:00:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/get_stocks.py', '--cfg-path', '/tmp/tmpwj_kbpi1', '--error-file', '/tmp/tmpzztc0krh']
[2024-09-15 20:02:14,260] {standard_task_runner.py:80} INFO - Job 11: Subtask BRONZE_GOOG
[2024-09-15 20:02:14,287] {task_command.py:370} INFO - Running <TaskInstance: BRONZER_GET_STONCKS.BRONZE_GOOG scheduled__2024-01-02T00:00:00+00:00 [running]> on host kaiwang.
[2024-09-15 20:02:14,325] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=BRONZER_GET_STONCKS
AIRFLOW_CTX_TASK_ID=BRONZE_GOOG
AIRFLOW_CTX_EXECUTION_DATE=2024-01-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-01-02T00:00:00+00:00
[2024-09-15 20:02:15,103] {python.py:173} INFO - Done. Returned value was: None
[2024-09-15 20:02:15,108] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=BRONZER_GET_STONCKS, task_id=BRONZE_GOOG, execution_date=20240102T000000, start_date=20240915T230214, end_date=20240915T230215
[2024-09-15 20:02:15,158] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-09-15 20:02:15,189] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-15 20:05:54,839] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: BRONZER_GET_STONCKS.BRONZE_GOOG scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-09-15 20:05:54,843] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: BRONZER_GET_STONCKS.BRONZE_GOOG scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-09-15 20:05:54,843] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-09-15 20:05:54,843] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2024-09-15 20:05:54,843] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-09-15 20:05:54,854] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): BRONZE_GOOG> on 2024-01-02 00:00:00+00:00
[2024-09-15 20:05:54,856] {standard_task_runner.py:52} INFO - Started process 11849 to run task
[2024-09-15 20:05:54,859] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'BRONZER_GET_STONCKS', 'BRONZE_GOOG', 'scheduled__2024-01-02T00:00:00+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/get_stocks.py', '--cfg-path', '/tmp/tmp21ya9voi', '--error-file', '/tmp/tmpbd0kgk0o']
[2024-09-15 20:05:54,859] {standard_task_runner.py:80} INFO - Job 13: Subtask BRONZE_GOOG
[2024-09-15 20:05:54,887] {task_command.py:370} INFO - Running <TaskInstance: BRONZER_GET_STONCKS.BRONZE_GOOG scheduled__2024-01-02T00:00:00+00:00 [running]> on host kaiwang.
[2024-09-15 20:05:54,926] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=BRONZER_GET_STONCKS
AIRFLOW_CTX_TASK_ID=BRONZE_GOOG
AIRFLOW_CTX_EXECUTION_DATE=2024-01-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-01-02T00:00:00+00:00
[2024-09-15 20:05:54,927] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/usr/lib/python3.10/pathlib.py", line 1175, in mkdir
    self._accessor.mkdir(self, mode)
FileNotFoundError: [Errno 2] No such file or directory: '/datalake/GOOG'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kaiwang/Documents/airflow_celeby/venv/lib/python3.10/site-packages/airflow/decorators/base.py", line 179, in execute
    return_value = super().execute(context)
  File "/home/kaiwang/Documents/airflow_celeby/venv/lib/python3.10/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/kaiwang/Documents/airflow_celeby/venv/lib/python3.10/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/kaiwang/Documents/airflow_celeby/dags/get_stocks.py", line 21, in get_history
    (Path(file_path).parent).mkdir(parents=True, exist_ok=True)
  File "/usr/lib/python3.10/pathlib.py", line 1179, in mkdir
    self.parent.mkdir(parents=True, exist_ok=True)
  File "/usr/lib/python3.10/pathlib.py", line 1175, in mkdir
    self._accessor.mkdir(self, mode)
PermissionError: [Errno 13] Permission denied: '/datalake'
[2024-09-15 20:05:54,930] {taskinstance.py:1395} INFO - Marking task as FAILED. dag_id=BRONZER_GET_STONCKS, task_id=BRONZE_GOOG, execution_date=20240102T000000, start_date=20240915T230554, end_date=20240915T230554
[2024-09-15 20:05:54,939] {standard_task_runner.py:92} ERROR - Failed to execute job 13 for task BRONZE_GOOG ([Errno 13] Permission denied: '/datalake'; 11849)
[2024-09-15 20:05:54,951] {local_task_job.py:156} INFO - Task exited with return code 1
[2024-09-15 20:05:54,961] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-15 20:08:37,693] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: BRONZER_GET_STONCKS.BRONZE_GOOG scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-09-15 20:08:37,698] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: BRONZER_GET_STONCKS.BRONZE_GOOG scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-09-15 20:08:37,698] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-09-15 20:08:37,698] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2024-09-15 20:08:37,698] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-09-15 20:08:37,708] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): BRONZE_GOOG> on 2024-01-02 00:00:00+00:00
[2024-09-15 20:08:37,711] {standard_task_runner.py:52} INFO - Started process 12441 to run task
[2024-09-15 20:08:37,713] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'BRONZER_GET_STONCKS', 'BRONZE_GOOG', 'scheduled__2024-01-02T00:00:00+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/get_stocks.py', '--cfg-path', '/tmp/tmpiaeeymsi', '--error-file', '/tmp/tmpgnvv6lab']
[2024-09-15 20:08:37,713] {standard_task_runner.py:80} INFO - Job 62: Subtask BRONZE_GOOG
[2024-09-15 20:08:37,740] {task_command.py:370} INFO - Running <TaskInstance: BRONZER_GET_STONCKS.BRONZE_GOOG scheduled__2024-01-02T00:00:00+00:00 [running]> on host kaiwang.
[2024-09-15 20:08:37,778] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=BRONZER_GET_STONCKS
AIRFLOW_CTX_TASK_ID=BRONZE_GOOG
AIRFLOW_CTX_EXECUTION_DATE=2024-01-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-01-02T00:00:00+00:00
[2024-09-15 20:08:38,376] {python.py:173} INFO - Done. Returned value was: None
[2024-09-15 20:08:38,380] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=BRONZER_GET_STONCKS, task_id=BRONZE_GOOG, execution_date=20240102T000000, start_date=20240915T230837, end_date=20240915T230838
[2024-09-15 20:08:38,409] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-09-15 20:08:38,419] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-15 20:12:28,462] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: BRONZER_GET_STONCKS.BRONZE_GOOG scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-09-15 20:12:28,467] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: BRONZER_GET_STONCKS.BRONZE_GOOG scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-09-15 20:12:28,468] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-09-15 20:12:28,468] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2024-09-15 20:12:28,468] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-09-15 20:12:28,479] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): BRONZE_GOOG> on 2024-01-02 00:00:00+00:00
[2024-09-15 20:12:28,482] {standard_task_runner.py:52} INFO - Started process 13066 to run task
[2024-09-15 20:12:28,484] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'BRONZER_GET_STONCKS', 'BRONZE_GOOG', 'scheduled__2024-01-02T00:00:00+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/get_stocks.py', '--cfg-path', '/tmp/tmpweelf04l', '--error-file', '/tmp/tmp5je3rd25']
[2024-09-15 20:12:28,484] {standard_task_runner.py:80} INFO - Job 62: Subtask BRONZE_GOOG
[2024-09-15 20:12:28,513] {task_command.py:370} INFO - Running <TaskInstance: BRONZER_GET_STONCKS.BRONZE_GOOG scheduled__2024-01-02T00:00:00+00:00 [running]> on host kaiwang.
[2024-09-15 20:12:28,554] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=BRONZER_GET_STONCKS
AIRFLOW_CTX_TASK_ID=BRONZE_GOOG
AIRFLOW_CTX_EXECUTION_DATE=2024-01-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-01-02T00:00:00+00:00
[2024-09-15 20:12:28,554] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/usr/lib/python3.10/pathlib.py", line 1175, in mkdir
    self._accessor.mkdir(self, mode)
FileNotFoundError: [Errno 2] No such file or directory: '/Documents/airflow_celeby/datalake/GOOG'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.10/pathlib.py", line 1175, in mkdir
    self._accessor.mkdir(self, mode)
FileNotFoundError: [Errno 2] No such file or directory: '/Documents/airflow_celeby/datalake'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.10/pathlib.py", line 1175, in mkdir
    self._accessor.mkdir(self, mode)
FileNotFoundError: [Errno 2] No such file or directory: '/Documents/airflow_celeby'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kaiwang/Documents/airflow_celeby/venv/lib/python3.10/site-packages/airflow/decorators/base.py", line 179, in execute
    return_value = super().execute(context)
  File "/home/kaiwang/Documents/airflow_celeby/venv/lib/python3.10/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/kaiwang/Documents/airflow_celeby/venv/lib/python3.10/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/kaiwang/Documents/airflow_celeby/dags/get_stocks.py", line 21, in get_history
    (Path(file_path).parent).mkdir(parents=True, exist_ok=True)
  File "/usr/lib/python3.10/pathlib.py", line 1179, in mkdir
    self.parent.mkdir(parents=True, exist_ok=True)
  File "/usr/lib/python3.10/pathlib.py", line 1179, in mkdir
    self.parent.mkdir(parents=True, exist_ok=True)
  File "/usr/lib/python3.10/pathlib.py", line 1179, in mkdir
    self.parent.mkdir(parents=True, exist_ok=True)
  File "/usr/lib/python3.10/pathlib.py", line 1175, in mkdir
    self._accessor.mkdir(self, mode)
PermissionError: [Errno 13] Permission denied: '/Documents'
[2024-09-15 20:12:28,557] {taskinstance.py:1395} INFO - Marking task as FAILED. dag_id=BRONZER_GET_STONCKS, task_id=BRONZE_GOOG, execution_date=20240102T000000, start_date=20240915T231228, end_date=20240915T231228
[2024-09-15 20:12:28,572] {standard_task_runner.py:92} ERROR - Failed to execute job 62 for task BRONZE_GOOG ([Errno 13] Permission denied: '/Documents'; 13066)
[2024-09-15 20:12:28,577] {local_task_job.py:156} INFO - Task exited with return code 1
[2024-09-15 20:12:28,587] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-15 20:14:35,899] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: BRONZER_GET_STONCKS.BRONZE_GOOG scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-09-15 20:14:35,905] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: BRONZER_GET_STONCKS.BRONZE_GOOG scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-09-15 20:14:35,905] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-09-15 20:14:35,905] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2024-09-15 20:14:35,905] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-09-15 20:14:35,915] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): BRONZE_GOOG> on 2024-01-02 00:00:00+00:00
[2024-09-15 20:14:35,918] {standard_task_runner.py:52} INFO - Started process 13557 to run task
[2024-09-15 20:14:35,920] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'BRONZER_GET_STONCKS', 'BRONZE_GOOG', 'scheduled__2024-01-02T00:00:00+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/get_stocks.py', '--cfg-path', '/tmp/tmpk14ucdcc', '--error-file', '/tmp/tmp91xflds0']
[2024-09-15 20:14:35,921] {standard_task_runner.py:80} INFO - Job 62: Subtask BRONZE_GOOG
[2024-09-15 20:14:35,949] {task_command.py:370} INFO - Running <TaskInstance: BRONZER_GET_STONCKS.BRONZE_GOOG scheduled__2024-01-02T00:00:00+00:00 [running]> on host kaiwang.
[2024-09-15 20:14:35,992] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=BRONZER_GET_STONCKS
AIRFLOW_CTX_TASK_ID=BRONZE_GOOG
AIRFLOW_CTX_EXECUTION_DATE=2024-01-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-01-02T00:00:00+00:00
[2024-09-15 20:14:36,568] {python.py:173} INFO - Done. Returned value was: None
[2024-09-15 20:14:36,572] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=BRONZER_GET_STONCKS, task_id=BRONZE_GOOG, execution_date=20240102T000000, start_date=20240915T231435, end_date=20240915T231436
[2024-09-15 20:14:36,617] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-09-15 20:14:36,628] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-15 20:15:37,738] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: BRONZER_GET_STONCKS.BRONZE_GOOG scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-09-15 20:15:37,743] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: BRONZER_GET_STONCKS.BRONZE_GOOG scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-09-15 20:15:37,743] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-09-15 20:15:37,743] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2024-09-15 20:15:37,743] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-09-15 20:15:37,754] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): BRONZE_GOOG> on 2024-01-02 00:00:00+00:00
[2024-09-15 20:15:37,756] {standard_task_runner.py:52} INFO - Started process 13716 to run task
[2024-09-15 20:15:37,759] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'BRONZER_GET_STONCKS', 'BRONZE_GOOG', 'scheduled__2024-01-02T00:00:00+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/get_stocks.py', '--cfg-path', '/tmp/tmpspha7t73', '--error-file', '/tmp/tmpmuve72kc']
[2024-09-15 20:15:37,760] {standard_task_runner.py:80} INFO - Job 62: Subtask BRONZE_GOOG
[2024-09-15 20:15:37,792] {task_command.py:370} INFO - Running <TaskInstance: BRONZER_GET_STONCKS.BRONZE_GOOG scheduled__2024-01-02T00:00:00+00:00 [running]> on host kaiwang.
[2024-09-15 20:15:37,837] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=BRONZER_GET_STONCKS
AIRFLOW_CTX_TASK_ID=BRONZE_GOOG
AIRFLOW_CTX_EXECUTION_DATE=2024-01-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-01-02T00:00:00+00:00
[2024-09-15 20:15:38,378] {python.py:173} INFO - Done. Returned value was: None
[2024-09-15 20:15:38,383] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=BRONZER_GET_STONCKS, task_id=BRONZE_GOOG, execution_date=20240102T000000, start_date=20240915T231537, end_date=20240915T231538
[2024-09-15 20:15:38,417] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-09-15 20:15:38,427] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
